server/jetstream_cluster_1_test.go

```diff
@@ -6696,13 +6698,32 @@ func TestJetStreamConsumerAckOutOfBounds(t *testing.T) {
 	msgs, err := sub.Fetch(1)
 	require_NoError(t, err)
 	require_Equal(t, len(msgs), 1)
-	msgs[0].AckSync()
+	ackSubject := msgs[0].Reply
+
+	var resp JSApiConsumerAckResponse
+
+	// first ack
+	replyMsg, err := nc.Request(ackSubject, nil, 5000*time.Millisecond)
+	require_NoError(t, err)
+	require_NoError(t, json.Unmarshal(replyMsg.Data, &resp))
+	require_True(t, resp.Success)
+
+	// check for error if trying to ack again
+	resp = JSApiConsumerAckResponse{}
+	replyMsg, err = nc.Request(ackSubject, nil, 250*time.Millisecond)
+	require_NoError(t, err)
+	require_NoError(t, json.Unmarshal(replyMsg.Data, &resp))
+	require_False(t, resp.Success)
+	require_True(t, resp.Error.ErrCode == uint16(JSConsumerMsgNotPendingAckErr))
```

Comment:
Same thing here (and below) about the more explicit error messages.

Reply:
Also done